coding/python_testing.rst

```diff
@@ -183,7 +183,7 @@ including :lclass:`~lsst.utils.tests.MemoryTestCase`:
 .. literalinclude:: unit_test_snippets/unittest_runner_example.py
    :linenos:
    :language: python
-   :emphasize-lines: 6, 16, 21
+   :emphasize-lines: 12, 17, 21
```

Comment:
They are the lines relevant to memory leak testing. This is the memory testing section.